Dockerfile

```diff
@@ -13,9 +13,9 @@ RUN \
     && mkdir runner \
     && tar xzf "actions-runner-linux-x64-${ACTIONS_VERSION}.tar.gz" --directory ./runner
 
-FROM ubuntu:22.10
+FROM ubuntu:23.04
 
-RUN addgroup --gid 1000 "runner" && adduser --uid 1000 --ingroup "runner" --shell /bin/bash "runner" \
+RUN groupadd --gid 1234 "runner" && useradd -g "runner" --shell /bin/bash "runner" \
```

Comment:
For the group ID, did 1000 not work on ubuntu 23? I think they were following the standard for this:

> By default, the new group gets created with a group id higher than the GID_MIN value defined in /etc/login.defs. On most Linux system, this value is 1000.
> In other words, you get one of the first available GID after 1000.

1000 might not work for a container you are logged into vs being built by Docker.

Reply:
Per our Slack conversation, we'll just skip specifying the group ID since we don't need to reference it anywhere else